py/desisim/scripts/quickquasars.py

```diff
@@ -494,6 +492,9 @@ def simulate_one_healpix(ifilename,args,model,obsconditions,decam_and_wise_filte
             # restore random state to get the same random numbers later
             # as when we don't insert BALs
             np.random.set_state(rnd_state)
+            meta_bal['TARGETID'] = metadata['MOCKID']
+            w = meta_bal['TEMPLATEID']!=-1
+            meta_bal = meta_bal[:][w]
```

Comment:
Still there is more information that we probably want to the metadata, like the BAL index, which I'm not sure is the same as the balnicity, Do you know?  this can be added in another issue anyway.

Reply:
@alxogm, Yes it can be added later.